modules/services/mako.nix

```diff
@@ -312,5 +330,23 @@ in {
       ${optionalBoolean "ignore-timeout" cfg.ignoreTimeout}
       ${optionalString "group-by" cfg.groupBy}
     '';
+
+    systemd.user.services.mako = {
+      Unit = {
+        Description = "Lightweight Wayland notification daemon";
+        Documentation = "man:mako(1)";
+        PartOf = [ cfg.systemdTarget ];
+        After = [ cfg.systemdTarget ];
```

Comment:
Sorry, this should be set to `graphical-session.target`.
The part the user will want to change is the `Install.WantedBy` option.

Reply:
Ah right, sorry for that.